src/main/java/frc/robot/commands/ClimbUPCamand.java

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.ClimbSubsystem; 
+
+public class ClimbUPCamand extends CommandBase {
+  
+  private final ClimbSubsystem m_subsystem;
+  
+  public ClimbUPCamand(ClimbSubsystem s) {
+    // Use addRequirements() here to declare subsystem dependencies.
+    addRequirements(s);
+    m_subsystem = s;
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    m_subsystem.enableClimb(true, true);
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {}
+
+  // Called once the command ends or is interrupted.
+  @Override
+  public void end(boolean interrupted) {
+    m_subsystem.enableClimb(false, false);
+  }
+
+  // Returns true when the command should end.
+  @Override
+  public boolean isFinished() {
+    return false;
```

Comment:
Should we be checking that the enableClimb setreference is finished here? 
 ( I assume that would require another method in the subsystem. )
 
 Not seeing a sparkmax code example for using the position